Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -178,6 +178,10 @@ public void SearchValidationsRecursive(DependencyObject depObj)
                         {
                             ComboBox comboBox = (ComboBox)child;
                             bindingExpression = comboBox.GetBindingExpression(ComboBox.SelectedValueProperty);
+                            if(bindingExpression == null)
+                            {
+                                bindingExpression = comboBox.GetBindingExpression(ComboBox.TextProperty);
+                            }
                         }
```

Comment:
in which case we needed binding to the comboBox.TextProperty and not SelectedValueProperty?

Reply:
This is required in case of Validation sometimes it is not working for SelectedValue so added the binding on Text property